Resources/Prototypes/Entities/Objects/Devices/nuke.yml

```diff
@@ -20,6 +20,8 @@
         mass: 150
         layer:
         - SmallImpassable
+        mask:
+        - VaultImpassable
```

Comment:
These should probably be Impassable but I want to refactor collision groups at some stage anyway